.changeset/sour-forks-stare.md

```diff
@@ -0,0 +1,5 @@
+---
+'svelte': patch
+---
+
+chore: remove redundant hydration code
```

Comment:
This might not be an accurate description anymore now that it's adding rather than removing code. I don't care so much about the changeset, but just thought it was funny :laughing:

![Screenshot from 2023-11-29 20-19-53](https://github.com/sveltejs/svelte/assets/322311/5768af65-3a0f-4aed-8cf3-e4f5f07c8765)